Game/GameWorld.h

```diff
@@ -1,4 +1,5 @@
 #pragma once
+#include "Functionalities/TransformFunctionality.h"
```

Comment:
Do we need this include?

Reply:
Yes, because I turned TransformFunctionality into a template. So forward declaring it requires a bit more writing, and since it's not included in a lot of header files, I decided to just include it normally to make it more readable.